Release-schedule-and-other-information.md

```diff
@@ -8,34 +8,36 @@ Please see the rota below for the expected release dates. We expect QA testing t
 
```

Comment:
Team leads (above) are you and Chris, right?

Reply:
done